targets/PandABrick/const/SFP_MGT_pins.xdc

```diff
@@ -0,0 +1,8 @@
+
+# Tx and Rx Pins...
+
+#set_property PACKAGE_PIN V2 [get_ports ch1_gthrxp_in]
```

Comment:
is it worth keeping this file if it is not doing anything?

Reply:
Good spot, I hadn't realised they were commented. They should probably be either uncommented as for the other targets, or removed - we only need this file when not using the MGT, but for PandABrick the MGT is not optional, so its not strictly needed. I don't mind either way.

Reply:
Fixed in fix-up commit.